front/cypress/e2e/project_page_timeline.cy.ts

```diff
@@ -152,25 +152,25 @@ describe('New timeline project', () => {
       'not.exist'
     );
     cy.get('#e2e-project-phase-description-read-more-button')
-      .should('exist')
+      .should('be.visible')
       .click();
 
     cy.get('#e2e-project-phase-description-read-more-button').should(
       'not.exist'
     );
     cy.get('#e2e-project-phase-description-see-less-button')
-      .should('exist')
+      .should('be.visible')
```

Comment:
Ensuring the button is visible rather than DOM existence hopefully stabilizes the test.

At least, when I commented out all other tests, I had a similar flakiness issue with the "show more" button, which got resolved when I changed the test from "exists" to "be.visible".